schema/mzqc_schema.json

```diff
@@ -19,6 +19,18 @@
                     "type": "string",
                     "format": "date-time"
                 },
+                "readMe": {
```

Comment:
I'd maybe prefer "description" over "readMe" because that feels a bit less like an obligation imo.

Reply:
Ok